scheduler not mentioned as peerDependency in documentation

When using this package with yarn v2 (yarn berry), it throws while building:

```
Module not found: Error: jotai tried to access scheduler (a peer dependency) but it isn't provided by your application; this makes the require call ambiguous and unsound.
```

I can resolve it by adding [scheduler](https://www.npmjs.com/package/scheduler) to my dependencies.

To solve the problem from your side, I see 2 possibilities:
- You could add an installation instruction, that mentions the need to install react and scheduler or 
- you could define scheduler as a dependency
- (or remove the scheduler-dependency if not needed ;-) )

--- a/readme.md
+++ b/readme.md
@@ -173,9 +173,12 @@ yarn add jotai react scheduler react-dom
 ```
 
 For React Native users:
+
 ```bash
 yarn add jotai react scheduler react-native
 ```
+
+For users who use workspaces, check out [this issue](https://github.com/pmndrs/jotai/issues/135) for some workarounds.
 
 ## More information
 

--- a/readme.md
+++ b/readme.md
@@ -164,6 +164,19 @@ function Controls() {
 
 ----
 
+## Installation notes for Yarn 2 users
+
+If you are using yarn v2 (yarn berry), you need to install all peer dependencies by yourself.
+
+```bash
+yarn add jotai react scheduler react-dom
+```
+
+For React Native users:
+```bash
+yarn add jotai react scheduler react-native
+```
+
 ## More information
 
 We will be organizing some more information later. Meanwhile, please see WIP materials.

